PamolaProject/PamolaProject/CMakeLists.txt

```diff
@@ -1,6 +1,12 @@
 
 add_definitions(-std=c++0x)
 
+file(DOWNLOAD
```

Comment:
We tried to use CMake list, but have some problems in the past. We had issues with including external libraries to our project. Thank you.